O3exp/pipeline/plotting.ini

```diff
@@ -418,11 +418,14 @@ sngl-stat = ${fit_by_template|sngl-stat}
 fit-function = ${fit_by_template|fit-function}
 ; don't try and fit everything down to zero stat value
 stat-fit-threshold = ${fit_by_template|stat-threshold}
+prune-number = 1 
+; note this pruning not the same style as in fit_by_template
+; default 0.1 prune window
 
 [plot_binnedhist-duration]
 bin-param = template_duration
 num-bins = 6
-; bin spacing is always log
+; default bin spacing is log
```

Comment:
as above probably better to remove comments about defaults in this section (eg in case the defaults change at some point)